### ISSUE TITLE
Gigasecond: Fix timezone sensitivity

### ISSUE THREAD

**Issue by jnv** [exercism/elm#228]
The tests for Gigasecond side exercise seem to be timezone-sensitive, e.g. I am in CET and I get the following error:

```
✗ 1977-06-13

    <Thu Feb 19 2009 02:46:40 GMT+0100 (CET)>
    ╷
    │ Expect.equal
    ╵
    <Thu Feb 19 2009 01:46:40 GMT+0100 (CET)>
```

When I set timezone like this, the test passes:

```
TZ=UTC elm-test
```

Would it be possible to set timezone in test file to prevent these errors?

**Comment by tuxagon** [comment]
@jnv nice catch! Thanks for reporting this. I'll look into it sometime soon and update accordingly.

**Comment by N-Parsons** [comment]
Looks like this is fixed in #230 with the move to Posix time rather than Date.

**Comment by mpizenberg** [comment]
Fixed indeed
